awx/main/conf.py

```diff
@@ -831,6 +831,15 @@
     category_slug='system',
 )
 
+register(
+    'HOST_METRIC_SUMMARY_TASK_LAST_TS',
+    field_class=fields.DateTimeField,
+    label=_('Last computing date of HostMetricSummaryMonthly'),
+    allow_null=True,
+    category=_('System'),
+    category_slug='system',
```

Comment:
anyway, althought it's not intended to have these settings in the UI, it's perfectly safe if user changes it (i.e. with recommendation of RH support), now it can change the interval between two runs